release_scripts/Packages.Microsoft/supported_repos_testing.json

```diff
@@ -0,0 +1,280 @@
+[
```

Comment:
Is this coming from a package? If so, can we just download package and avoid checking this is? My concern is these values can change.

Reply:
These values won't change—they map to repos in Packages.Microsoft. If you query the Repo CLI you can get this information.

The key bit of data is the repository ID, which needs to be in the root config when uploading a package. Without some sort of automation, you'd need to look up the repository ID for each Linux repo you're uploading to, and edit the root config manually.